tests/expected/lsfd/mkfds-tcp6

```diff
@@ -0,0 +1,4 @@
+    3 TCPv6   SOCK state=listen laddr=[::1]:34567                             listen    stream              1         ::1          :: [::1]:34567     34567      [::]:0         0
```

Comment:
Note: TCPv6 uses a different naming scheme than INET6

Reply:
Do you mean the name "TCP6" is better than "TCPv6"?

Reply:
It would be more consistent and I like it a bit more.
Nothing important though.

Reply:
Oh, I see. The name TCPv6 comes from 
```
# getfattr -d -m 'system.*' /proc/1137/fd/4;echo
getfattr: Removing leading '/' from absolute path names
# file: proc/1137/fd/4
system.sockprotoname="TCPv6"
```